main.go

```diff
@@ -857,3 +883,30 @@ func readZipFile(zf *zip.File) ([]byte, error) {
 	defer f.Close()
 	return ioutil.ReadAll(f)
 }
+
+func readS3EndpointCA(endpointCA string) ([]byte, error) {
```

Comment:
My suggestion would be to make this more explicit:

- Check if its a file and log
- Check if its base64 encoded and log

I would also add a check for a valid certificate before continuing, possibly outside this function